generators/app/templates/Components/functional/styles.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+import styled from 'styled-components/native'
+import variables from 'Olio/App/Styles/Variables'
+
```

Comment:
While we are here shall we add a flow types for types_first setting that is coming. Not sure if View has a flow type definition so try;

```
type ViewProps= typeof View

type ContainerProps = {
   color: string,
   ...ViewProps,
}

export const Container: ContainerProps ...
```

I'm not sure if would need to do `ComponentType<ContainerProps>`. Or could flag in that types first stylesheet ticket to update the generator then?

Reply:
Great idea will get that added

Reply:
I am going to ask that you do this in a follow up PR (since I'm guessing we'd need to update both templates?)

Reply:
https://github.com/OLIOEX/generator-olio/issues/17